selfdrive/ui/qt/home.cc

```diff
@@ -74,8 +78,8 @@ void HomeWindow::mousePressEvent(QMouseEvent* e) {
   }
 
   // Vision click
-  if (ui_state->scene.started && (e->x() >= ui_state->viz_rect.x - bdr_s)) {
-    ui_state->sidebar_collapsed = !ui_state->sidebar_collapsed;
+  if (ui_state->scene.started && (e->x() >= sidebar->width())) {
```

Comment:
this doesn't account for whether the sidebar is collapsed, there's probably a nice way to do this in Qt